src/main/java/az/edu/turing/domain/entity/BookingEntity.java

```diff
@@ -0,0 +1,72 @@
+package az.edu.turing.domain.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+public class BookingEntity {
+
+    private String id;
+    private FlightEntity flight;
+    private final List<PassengerEntity> passengers;
+    private boolean isActive;
+
+    public BookingEntity(FlightEntity flight) {
+        this.id = UUID.randomUUID().toString().substring(0, 8);
```

Comment:
What do we gain by doing this?

Reply:
UUİD provides random string. But size of it is too long, so I decided to use only first 8 of it.

Reply:
Leave it, as it is :)